signer/migrations/0003__create_tables.sql

```diff
@@ -196,6 +196,8 @@ CREATE TABLE sbtc_signer.sweep_transactions (
     amount BIGINT NOT NULL,
     -- The fee paid for the transaction.
     fee BIGINT NOT NULL,
+    -- The _vsize_ of the transaction in bytes.
+    vsize INTEGER NOT NULL,
```

Comment:
We should make this a `BIGINT` since the max is `u32::MAX`. Hmmm, it feels like a waste of space, but it is more correct.

Reply:
Hmm, yeah. And we should probably cap it ourselves anyway.